sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1277,6 +1286,15 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
     loss_ : float
         The current loss computed with the loss function.
 
+    best_loss_ : float
+        The minimum loss reached by the solver throughout fitting.
+
+    loss_curve_ : list, length n_iter_
```

Comment:
same remarks than for MLPClassifier

Reply:
Alright, thank you I'll make the changes!